db/migrate/20170907144901_add_columns_to_training_data_measurement.rb

```diff
@@ -0,0 +1,8 @@
+class AddColumnsToTrainingDataMeasurement < ActiveRecord::Migration[5.0]
+  def change
+    add_column :training_data_measurements, :x, :double
```

Comment:
Define x,y,z as float instead as double.

Reply:
Why is that?

Reply:
http://www.ilikebigbits.com/blog/2017/6/1/float-or-double 

Summary: 

"If you have plenty of RAM and neither execution speed or battery drainage is a problem for you - you can stop reading now and just use double. Good bye and have a nice day!

If you are memory constrained, then the reason to use float over double might be as simple as "it takes half as much space"."

Reply:
And how can I fix this problem? Can I remove the columns using RemoveColumnsToTrainingDataMeasurement?x y z .       And then  AddNewColumnsToTrainingDataMeasurement x:float y: float: z:float 

Is that okay?

Reply:
That's a way to do it. But then we would have tons of "migration corrections", which I don't find really clean. I would suggest to rollback and remove this migration and then create a new one with the correct type of field. I know we already have used that method in the past:

RemoveColumnsToTrainingDataMeasurement?x y z . And then AddNewColumnsToTrainingDataMeasurement x:float y: float: z:float

But the idea now is to do it as clean as possible and following certain good programming practices rules to avoid future bugs or errors in our code.